Editor/Shading/ShowIfMaterialPropertyDrawer.cs

```diff
@@ -17,51 +17,51 @@ namespace StansAssets.Foundation.Editor.Shading
 	///     [ShowIf(_BASE_COLOR_ON)] _BaseColor("", Color) = (1,1,1,1)
     /// </code>
     /// </example>
-    public class ShowIfDrawer : MaterialPropertyDrawer
+    public class ShowIfMaterialPropertyDrawer : MaterialPropertyDrawer
```

Comment:
It'll be better to rename the file itself so that you can use the short version of the attribute as [ShowIf]